TAF/testScenarios/integrationTest/UC_CORS_configuration/cors_configuration.robot

```diff
@@ -0,0 +1,107 @@
+             ...      AND  Restart Services  core-metadata
+Suite Teardown  Run Keywords  Set Service.CORSConfiguration.EnableCORS to false For core-metadata On Consul
+                ...      AND  Restart Services  core-metadata
+                ...      AND  Run Teardown Keywords
+Force Tags      MessageBus=redis
+
+*** Variables ***
+${SUITE}  CORS Configuration
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/cors_configuration.log
+${coreMetadataUrl}  ${URI_SCHEME}://${BASE_URL}:${CORE_METADATA_PORT}
+
+*** Test Cases ***
+CORS001-Enable CORS without Origin header
+    ${header_keys}  Create List  Access-Control-Request-Method
+    ${header_values}  Create List  GET
+    ${unexpected_headers}  Create List  Access-Control-Expose-Headers  Access-Control-Allow-Origin
+                         ...     Access-Control-Allow-Credentials  Vary
+    When Send GET Request With Headers  ${header_keys}  ${header_values}
+    Then Header Should Not Contain  ${unexpected_headers}
+
+CORS002-Enable CORS and receive a prefligt request
+    ${header_keys}  Create List  Access-Control-Request-Method  Origin
+    ${header_values}  Create List  GET  http://localhost
+    ${expected_headers}  Create List  Vary  Access-Control-Allow-Headers  Access-Control-Allow-Methods
+                         ...   Access-Control-Allow-Origin  Access-Control-Allow-Credentials  Access-Control-Max-Age
+    ${unexpected_headers}  Create List  Access-Control-Expose-Headers
+    When Send OPTIONS Request With Headers  ${header_keys}  ${header_values}
+    Then Header Should Contain  ${expected_headers}
+    And Header Should Not Contain  ${unexpected_headers}
+
+CORS003-Enable CORS and receive an actual request
+    ${header_keys}  Create List  Access-Control-Request-Method  Origin
+    ${header_values}  Create List  GET  http://localhost
+    ${expected_headers}  Create List  Access-Control-Expose-Headers  Access-Control-Allow-Origin
+                         ...     Access-Control-Allow-Credentials  Vary
+    When Send GET Request With Headers  ${header_keys}  ${header_values}
+    Then Header Should Contain  ${expected_headers}
+
+CORS004-Enable CORS and receive a prefligt request with CORSAllowCredentials=false
+    [Setup]  Run Keywords  Set Service.CORSConfiguration.CORSAllowCredentials to false For core-metadata On Consul
+             ...      AND  Restart Services  core-metadata
+    ${header_keys}  Create List  Access-Control-Request-Method  Origin
+    ${header_values}  Create List  GET  http://localhost
+    ${expected_headers}  Create List  Vary  Access-Control-Allow-Headers  Access-Control-Allow-Methods
+                         ...   Access-Control-Allow-Origin  Access-Control-Max-Age
+    ${unexpected_headers}  Create List  Access-Control-Expose-Headers
+    When Send OPTIONS Request With Headers  ${header_keys}  ${header_values}
+    Then Header Should Contain  ${expected_headers}
+    And Header Should Not Contain  ${unexpected_headers}
```

Comment:
it should allow either case
`Access-Control-Expose-Headers doesn't exist`
or
`Access-Control-Expose-Headers exists with the value false`

Reply:
`Access-Control-Expose-Headers exists with the value false` won't happen in our service